test/integration/scheduler/podsready/scheduler_test.go

```diff
@@ -370,18 +374,67 @@ var _ = ginkgo.Describe("SchedulerWithWaitForPodsReady", func() {
 			})
 		})
 	})
+
+	var _ = ginkgo.Context("Requeuing timestamp set to Creation", func() {
+
+		ginkgo.BeforeEach(func() {
+			podsReadyTimeout = 3 * time.Second
+			requeuingTimestamp = config.CreationTimestamp
+		})
+
+		ginkgo.It("Should prioritize workloads submitted earlier", func() {
+			localQueueName := "eviction-creation-lq"
```

Comment:
Why do we need to define a new lq? Should we use `prodQueue`?

Reply:
Good question, removing